src/tools/compiletest/src/runtest.rs

```diff
@@ -3431,11 +3431,23 @@ impl<'test> TestCx<'test> {
         let build_root = self.config.build_base.parent().unwrap().parent().unwrap();
         let build_root = cwd.join(&build_root);
 
-        let tmpdir = cwd.join(self.output_base_name());
-        if tmpdir.exists() {
-            self.aggressive_rm_rf(&tmpdir).unwrap();
+        // We construct the following directory tree for each rmake.rs test:
+        // ```
+        // base_dir/
+        //     rmake.exe
+        //     scratch/
```

Comment:
looks like this comment is out of date now with the rmake_out change?

Reply:
I updated the PR description but I forget to update the comment, thanks, I'll send a follow-up PR.

Reply:
Opened https://github.com/rust-lang/rust/pull/125896